app/workers/github_repos/repo_sync_worker.rb

```diff
@@ -26,7 +26,9 @@ def perform(repo_id)
           stargazers_count: fetched_repo.stargazers_count,
           info_hash: fetched_repo.to_hash,
         )
-        repo.user&.touch(:github_repos_updated_at)
+        if repo.user && repo.user.github_repos_updated_at < 30.minutes.ago
```

Comment:
**suggestion (non-blocking):** I don't know about others, but I find the comparison operators always a bit hard when it comes to dates/times in Rails, so I use the more explicit versions instead:

```suggestion
        if repo.user && repo.user.github_repos_updated_at.before?(30.minutes.ago)
```

Reply:
Thanks @citizen428 ! In the back of my mind, I knew something like this would exist in Ruby.